web_api/national_park/park.py

```diff
@@ -0,0 +1,27 @@
+# This code defines a Park class that has several attributes representing information about a park, such as its code, full name, description, latitude, longitude, email, phone, entrance fees, entrance passes, and operating hours.
+# The class also has a method that returns a list of all the parks in the database.
+
+
+# The __init__ method is the constructor of the class that initializes the first two attributes (park_code and full_name) 
+# with the values provided as arguments, and sets the remaining attributes to None.
+
+class Park():    # The park class has attributes     
+    def __init__(self, park_code, full_name): # the init method initializes the first two attributes with given
+        # parameters and sets the rest to None.
+
+        self.park_code = park_code
+        self.full_name = full_name
+        self.description = None
+        self.lat = None
+        self.lon = None
+        self.email = None
+        self.phone = None
+        self.entrance_fees = None
+        self.entrance_passes =  None
+        self.operating_hours = None
+
```

Comment:
please add 
```
self.forecast: dict = {}
```
so we dont need to have a separate park weather data class thanks!